packages/components/atoms/f-button/test-utils/component-objects/f-button--action.component.js

```diff
@@ -1,6 +1,6 @@
-const Page = require('@justeat/f-wdio-utils/src/base.page');
+import BasePage from '@justeat/f-wdio-utils';
 
-module.exports = class ActionButton extends Page {
+class ActionButton extends BasePage.Page {
```

Comment:
@tommcclean Is there a way this could be refactored to do the following that I'm not aware about?


```
import BasePage from '@justeat/f-wdio-utils';

class ActionButton extends BasePage {
```